app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends, cafe: Cafe):
```

Comment:
Your function won't pass this test:
```
(
    [
        {
            "name": "Alisa",
            "vaccine": {
                "name": "Pfizer",
                "expiration_date": datetime.date.today(),
            },
            "wearing_a_mask": False,
        },
        {
            "name": "Bob",
            "wearing_a_mask": True,
        },
    ],
    Cafe("KFC"),
    "All friends should be vaccinated",
),
```
I described this case in our slack chat, you can check it there

>Often, if an error occurs that one of the visitors is not wearing a mask, you will immediately return a message that several friends should buy a mask for themselves.
But this is wrong because if one of the following friends is not vaccinated, you need to return an error about vaccination.